synapseclient/.synapseConfig

```diff
@@ -4,19 +4,24 @@
 
 ## Used for logging in to Synapse. See https://python-docs.synapse.org/tutorials/authentication/
```

Comment:
We will need to make sure that we update this tutorial:

- https://python-docs.synapse.org/tutorials/authentication/

As well as consider how the CLI `synapse config` command will change as a result of these changes:

- https://python-docs.synapse.org/tutorials/command_line_client/#config